app/models/learning_hour.rb

```diff
@@ -30,6 +30,7 @@ def occurred_at_not_in_future
 #  id                    :bigint           not null, primary key
 #  duration_hours        :integer          not null
 #  duration_minutes      :integer          not null
+#  learning_type         :integer          default(5)
```

Comment:
Should this change be here?

Reply:
It's from merging main into the branch. Can back it out. Looks like there won't be merge conflicts.